young_yacks/game/game.py

```diff
@@ -1,3 +1,16 @@
-class Game:
-    def __init__(self):
-        self.player = Player()
+from .models import Environment, CowBreed, Game
+
+
+def init_game():
```

Comment:
I would maybe consider it making a `[classmethod](https://docs.python.org/3.9/library/functions.html#classmethod)`

Reply:
I was wondering that, too.  On Game, right?  I'm still conflicted about it, actually--putting parameter values in the models file seems like a big step, but let's try it out.  I'll add it in, and we can go back if necessary.